app/models/pageflow/entry.rb

```diff
@@ -43,6 +43,7 @@ class PasswordMissingError < StandardError
       create_draft!(home_button_enabled: theming.home_button_enabled_by_default)
       draft.storylines.create!(configuration: {main: true})
       theming.widgets.copy_all_to(draft)
+      theming.copy_default_meta_tags(draft)
```

Comment:
Ah ok, I actually liked the `copy_defaults_to` method you had before, just would have made these two calls the method body instead of inlining them. But this is really getting nitpicky.